src/stream.rs

```diff
@@ -769,6 +769,11 @@ impl RecvBuf {
         Ok(())
     }
 
+    /// Returns the cumulative offset of data buffered.
+    pub fn cumulative_off(&self) -> u64 {
```

Comment:
This name sounds wrong. It's not really "cumulative" as there's just one offset, so maybe just call it `off()`?

Reply:
as per discussion, renamed the old `off()` method to `off_front()` and `cumulative_off()` to `off_back()`.